x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -3,18 +3,62 @@
 
 beta::[This functionality is in alpha and is subject to change. The design and code is less mature than official GA features and is being provided as-is with no warranties. Alpha features are not subject to the support SLA of official GA features.]
 
-Before you start {es} for the first time, run the `elasticsearch-security-config`
-tool. This tool generates certificates and keys for the transport and HTTP
-layers of {es}, and writes the TLS configuration settings to `elasticsearch.yml`.
+When you start {es} for the first time, passwords are automatically generated
```

Comment:
One still needs to run the `elasticsearch-security-config` to get TLS autoconfigured, I think we want to retain this step and just add that the part about passwords.